hack/verify-boilerplate.sh

```diff
@@ -17,8 +17,7 @@
 set -o errexit
```

Comment:
why even have this file? Just replace the appropriate lines in `.pre-commit-config.yaml`

Reply:
I think it's better to keep this file and change it in the file because in this way, I don't need to think about where this file is used, and replace it everywhere.

Reply:
it's used in a single place --> pre-commit hooks. 

(also you could grep the whole repo and verify that)

Reply:
Yeah, currently, yes. But I think this is more maintainable, I don't need to even grep and verify if there are multiple places using this file, right?

Reply:
Also, this provides you a chance to verify boilerplate by just `./hack/verify-boilerplate.sh`?

Reply:
when was the last time you did that? 

Deleting code is magic!

Reply:
Also you could c/p the command from `.pre-commit-config.yaml` as well

Reply:
any update on this?

Reply:
fixed